### PR TITLE
CIV-15222 Update View Bundle and Trial Arrangements task List when Bundle is Restitched

### PR DESCRIPTION
**Pull request by kannan-v-hmcts**
PR for https://tools.hmcts.net/jira/browse/CIV-15222 & https://tools.hmcts.net/jira/browse/CIV-15382

### CODE REVIEW

**Review by m-meulendijks-v1 — APPROVED** [review]

File: src/main/resources/db/migration/V2024_09_20_1100__CIV-15222_bundle_restitch_task_list_scenario.sql [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/**
```
Comment (ss-evoco) [inline src/main/resources/db/migration/V2024_09_20_1100__CIV-15222_bundle_restitch_task_list_scenario.sql:1]: If these need running separately from the rest in the folder (for a separate release) then I think they should be in a new folder now.

File: src/main/resources/db/migration/V2024_09_23_1200__CIV-15382_bundle_restitch_remove_full_stop.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
```
Comment (ss-evoco) [inline src/main/resources/db/migration/V2024_09_23_1200__CIV-15382_bundle_restitch_remove_full_stop.sql:1]: Same with this.

**Review by ss-evoco — CHANGES_REQUESTED** [review]

File: src/main/resources/db/migration/V2024_09_20_1100__CIV-15222_bundle_restitch_task_list_scenario.sql (hunk 1 above)
Comment (kannan-v-hmcts) [reply src/main/resources/db/migration/V2024_09_20_1100__CIV-15222_bundle_restitch_task_list_scenario.sql:1]: HI Sean. Thanks for your review. As far as I know and I could be wrong as well, I could not find any config that tell the flyway DB to look for scripts in a specific folder so I am assuming it takes it from the default folder ie classpath:db/migration and the migration is enabled/disabled using REFERENCE_DATABASE_MIGRATION. At the moment its enabled in preview but not in demo/prod. its enabled once when a release happens and disabled again.

Comment (kannan-v-hmcts) [reply src/main/resources/db/migration/V2024_09_20_1100__CIV-15222_bundle_restitch_task_list_scenario.sql:1]: @dharmendrak Could you please confirm if this is right ?

**Review by ss-evoco — APPROVED** [review]

File: src/main/resources/db/migration/V2024_09_23_1200__CIV-15382_bundle_restitch_remove_full_stop.sql (hunk 2 above)
Comment (ss-evoco) [reply src/main/resources/db/migration/V2024_09_23_1200__CIV-15382_bundle_restitch_remove_full_stop.sql:1]: Agreed to do this in a later, separate ticket.

**Pull request merged**